docs/usage/node-local-dns.md

```diff
@@ -31,6 +31,9 @@ All that needs to be done to enable the usage of the `node-local-dns` feature is
 It is worth noting that: 
 
 - When migrating from IPVS to IPTables, existing pods will continue to leverage the node-local-dns cache. 
-- When migrating from IPtables to IPVS, only newer pods will be switched to the node-local-dns cache. 
+- When migrating from IPtables to IPVS, only newer pods will be switched to the node-local-dns cache.
+- The annotation will take effect during the next shoot reconciliation. This happens automatically once per day in the maintenance period (unless you have disabled it). 
+- During the reconfiguration of the node-local-dns there might be a short disruption in terms of domain name resolution depending on the setup. Usually, dns requests are repeated for some time as udp is an unreliable protocol, but that strictly depends on the application/way the domain name resolution happens. Its recommended to let the shoot be reconciled during the next maintenance period. 
```

Comment:
```suggestion
- During the reconfiguration of the node-local-dns there might be a short disruption in terms of domain name resolution depending on the setup. Usually, dns requests are repeated for some time as udp is an unreliable protocol, but that strictly depends on the application/way the domain name resolution happens. It is recommended to let the shoot be reconciled during the next maintenance period. 
```

Reply:
Thanks @ScheererJ  -- I've updated the commit with the change requested.